src/OnlineRetailPortal.Mock/Models/AddProductRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Mock
+{
+    public class AddProductRequest
```

Comment:
why do we need the same classes in mock  library ?

Reply:
we don't need rqrs in mock again. But we are yet discuss that with atul. Will do that today